src/vector/v.maxent.swd/v.maxent.swd.py

```diff
@@ -424,24 +513,20 @@ def main():
                     file=bgrtmp,
                     quiet=True,
                 )
-        grass.run_command(
-            "g.remove", type="vector", name=specname, flags="f", quiet=True
-        )
 
-    # Combine csv files
-    filenames = bgrdir + "/prespoints"
-    filenames = [filenames + str(i) for i in range(len(specs))]
-    with open(specout, "w") as outfile:
-        for fname in filenames:
-            with open(fname) as infile:
-                outfile.write(infile.read().rstrip() + "\n")
+        # Combine species swd files
+        filenames = os.path.join(bgrdir, "prespoints")
+        filenames = [filenames + str(i) for i in range(len(specs))]
+        with open(specout, "w") as outfile:
+            for fname in filenames:
+                with open(fname) as infile:
+                    outfile.write(infile.read().rstrip() + "\n")
```

Comment:
Could`nt you just append? That would reduce I/O

Reply:
Not sure for now how best to implement that. Patching the point layers would be another way, but possibly slower. I am leaving it for now, but have put it on the list of possible improvements.

Reply:
I thought of opening output file in append mode:
https://stackoverflow.com/questions/4706499/how-do-i-append-to-a-file#4706520

Reply:
Ah, that simple :-). Learned (again) something new. Thanks.. changed